hack/generate-cr.sh

```diff
@@ -146,7 +149,18 @@ spec:
         username: userPrincipalName
         idAttr: DN
         emailAttr: userPrincipalName
-        nameAttr: cn          
+        nameAttr: cn
+  - type: oidc
+    id: oidc
+    name: oidc
+    oidc: 
+      issuer: "https://${NAME}-${NS}.${APPS}"
```

Comment:
I'm confused by what we're representing here. It looks like the dex server will be pointing at itself as an OIDC provider. Shouldn't we be asking for the user to provide the OIDC Issuer URL just as they provide a client id and secret?

Reply:
you're right.. user will provide issuer URL